docs/using.rst

```diff
@@ -801,6 +801,27 @@ Example usage for DNS-01 (Cloudflare API v4) (for example purposes only, do not
 
 .. _lock-files:
 
+Changing the ACME Server
+========================
+
+By default, Certbot uses Let's Encrypt's initial production server at
+https://acme-v01.api.letsencrypt.org/. You can tell Certbot to use a
+different CA by providing ``--server`` on the command line or in a
+:ref:`configuration file <config-file>` with the URL of the server's
+ACME directory. For example, to test against Let's Encrypt's ACMEv2
+staging server, you would add ``--server
+https://acme-staging-v02.api.letsencrypt.org/directory`` to the command
+line. Certbot will automatically change the version of the ACME protocol
+used based on the contents served at the provided URL.
```

Comment:
Certbot will automatically ~change the~**select which** version of the ACME protocol to use based on the contents served at the provided URL.